plugin.js

```diff
@@ -16,13 +16,15 @@ const defaultOptions = {
 function verifyBearerAuthFactory (options) {
   const _options = Object.assign({}, defaultOptions, options)
   if (_options.keys instanceof Set) _options.keys = Array.from(_options.keys)
-  const { keys, errorResponse, contentType, bearerType, auth, addHook = true } = _options
-
+  const { keys, errorResponse, contentType, bearerType, auth, addHook = true, verifyErrorLogLevel = 'error' } = _options
+  const inValidLogLevelError = Error('invalid log level')
```

Comment:
Since `fastify` allow to pass custom logger and `pino` allow to add custom level. It should not be checked and restricted to 4 level only.
It would be fine to check if it is a `string`.

Reply:
Since this log level will be called as function, I concern if it will be leave unchecked as any string,

Does it have any check to make sure the passed string is safe?

Reply:
```js
Object.prototype.hasOwnProperty.call(fastify.log, level) && typeof fastify.log[level] === 'function'
```

Reply:
Thanks for solution, I've updated the code.

TODO

1. Unit test
2. the typescript definitions